SubjectBuilderBlobTrigger/exceptions.py

```diff
@@ -0,0 +1,19 @@
+""" exceptions.py: PostcodeSearchBuilder Custom Exceptions """
+
+
+class Error(Exception):
+    """ Base Exception Class """
+
+    pass
+
+
+class StopSubjectBuilderWarningException(Error):
```

Comment:
I'd remove code that isn't used like this. Just makes it harder to understand/maintain.